sklearn/tests/test_dummy.py

```diff
@@ -209,7 +209,6 @@ def test_classifier_exceptions():
 
 
 def test_mean_strategy_regressor():
-
```

Comment:
Can you revert all those changes in this file.
Long story short, we try to not modify code which is not linked to this PR (even PEP8)

Reply:
Sure.